discouraged

```diff
@@ -9062,7 +9062,10 @@
 "ispsubclN" is used by "pmapsubclN".
 "ispsubclN" is used by "psubcli2N".
 "ispsubclN" is used by "psubcliN".
+<<<<<<< HEAD
```

Comment:
After proper merging none of these lines <<<<<<< , ======= and >>>>>>> must appear in a file any more.  This is not a requirement of Metamath in particular, but applies to all text subject to git (and therefore GitHub).  I recommend to use the discouraged script (part of the Metamath bundle containing set.mm) to generate this file.